src/_premake_init.lua

```diff
@@ -1113,6 +1111,56 @@
 		linktimeoptimization("Default")
 	end)
 
+	api.deprecateValue("flags", "FatalWarnings", "Use `fatalwarnings { \"Compile\", \"Link\" }` instead.",
```

Comment:
Possibly, to avoid escaping with `\`
```suggestion
	api.deprecateValue('flags', 'FatalWarnings', 'Use `fatalwarnings { "Compile", "Link" }` instead.',
```
(apply to below code too)

Reply:
The only reason I'm against that is because we pretty much use quotes everywhere here and that breaks the convention. I could just use them around the warning message string, if we feel that strongly about escaping them